electronics/Arduino/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            Arduino
+version         1.8.8
+revision        0
+categories      electronics
+platforms       darwin
+maintainers     {gmail.com:giansalvo.gusinu} openmaintainer
+license         GPL-2
+
+description     Arduino Software (IDE) makes it easy to write code and upload it to the board. 
+
+long_description \
+    Arduino is an open-source electronics platform based on easy-to-use \
+    hardware and software. It's intended for anyone making interactive \
+    projects. The Arduino IDE runs on Windows, Mac OS X, and Linux. \
+    The environment is written in Java and based on Processing and other \
+    open-source software. This software can be used with any Arduino board.
+
+homepage        https://www.arduino.cc/
+
+master_sites    https://github.com/arduino/Arduino/archive/
+distfiles       ${version}.tar.gz
+
+
+checksums       rmd160  7fc2e6bdb8a1e8d55e8ba907a4ec3ea89b8e4a97 \
+                sha256  01bf72e54165a522fe80ae5454b7073994fd4139a326f2edc0bb61b1d226179a \
+                size    38704750
+
+depends_lib     port:apache-ant
+
+patchfiles
+
+post-patch {
+}
```

Comment:
If no patches are needed, just delete `pathfiles` and `post-patch` phase.

Reply:
ok